Testscripts/Windows/WORKLOAD-HB.ps1

```diff
@@ -313,24 +334,26 @@ install_package "fio iperf ethtool"
 			}
 		}
 
-		Start-WorkLoad "afterhb"
-
-		$postQueueSize = Run-LinuxCmd -ip $AllVMData[0].PublicIP -port $AllVMData[0].SSHPort -username $user -password $password -command "ethtool -S eth0 | grep -i tx_queue | grep bytes | wc -l" -runAsSudo
+		if (($isStorageWorkloadEnable -eq 1) -or ($isNetworkWorkloadEnable -eq 1)) {
+			Start-WorkLoad "afterhb"
+			if ($isNetworkWorkloadEnable -eq 1) {
+				$postQueueSize = Run-LinuxCmd -ip $AllVMData[0].PublicIP -port $AllVMData[0].SSHPort -username $user -password $password -command "ethtool -S eth0 | grep -i tx_queue | grep bytes | wc -l" -runAsSudo
 
-		if (($initialQueueSize -ne $postQueueSize) -and ($isNetworkWorkloadEnable -eq 1)) {
-			throw "Mismatching queue sizes of synthetic network interface eth0. Initially $initialQueueSize. $postQueueSize after resuming from hibernation"
+				if (($initialQueueSize -ne $postQueueSize) -and ($isNetworkWorkloadEnable -eq 1)) {
```

Comment:
($isNetworkWorkloadEnable -eq 1) it is a duplicated check

Reply:
Right. Fixed in 40fca2986bf49ccb35a348c44e5a2760d7a37dd6